src/language/tekster_en.ts

```diff
@@ -687,11 +692,13 @@ export default {
   'dokumentasjon.barnetilsynsbehov.beskrivelse':
     '<ul><li>Documentation from your doctor confirming that the child has medical, mental or major social problems and needs supervision. <li/><li>Documentation describing how much supervision your child needs, what kind of supervision, and how this affects your ability to be in occupational activity</li></ul>',
   'dokumentasjon.arbeidskontrakt.tittel':
-    'A work contract proving that you have been offered a job.',
+    'You must submit a work contract proving that you have been offered a job.',
```

Comment:
Ser det ikke ble innført her, men `work contract` er vel ingen god oversettelse 😅  Er `contract of employment` eller `employment contract` bedre?

Reply:
Google translate sier det, men enig i at det ikke egt virker helt rett hehe

Reply:
Bytter til Eiriks forslag